src/state/slices/accountSpecifiersSlice/selectors.ts

```diff
@@ -8,3 +9,16 @@ export const selectAccountSpecifierStrings = (state: ReduxState) =>
   state.accountSpecifiers.accountSpecifiers.map(accountSpecifier =>
     Object.entries(accountSpecifier)[0].join(':')
   )
+
+// returns a CAIP2-indexed object with all the `caip2:pubkeyish` accounts for that chainId
+export const selectPubkeyishByChainId = (state: ReduxState, chainId: CAIP2) =>
```

Comment:
Might be nice to have a bit more comments describing why this is needed and how used. Its not super clear based on the name